Formula/libxls.rb

```diff
@@ -0,0 +1,42 @@
+class Libxls < Formula
+  desc "Read binary Excel files from C/C++"
+  homepage "https://github.com/libxls/libxls"
+  url "https://github.com/libxls/libxls/releases/download/v1.6.2/libxls-1.6.2.tar.gz"
+  sha256 "5dacc34d94bf2115926c80c6fb69e4e7bd2ed6403d51cff49041a94172f5e371"
+  license "BSD-2-Clause"
+
+  def install
+    system "./configure", *std_configure_args, "--disable-silent-rules", "--program-prefix=lib"
```

Comment:
Why are we adding the prefix `lib`?

Reply:
The output from `configure` of `libxls` suggested it. It's because libxls wants to install a program called `xls2csv` and they recommend calling it `libxls2csv` because apparently there's other UNIX tool called `xls2csv`

Reply:
```
configure: catdoc users: avoid xls2csv conflicts, use ./configure --program-prefix=lib to get libxls2csv
```

^^ exact warning from the `configure`